composer.json

```diff
@@ -11,8 +11,8 @@
         ]
     },
     "require": {
-        "symfony/framework-bundle": ">=4.4",
+        "shopware/core": "*",
```

Comment:
@bjoern-flagbit if you ordered here the dependencies manually and wish them to be sorted, please add this section to the `composer.json`.
```
"config": {
    "sort-packages": true
}
```

Reply:
the main question is do we need the retry? or do I just remove the plugin:refresh ... because if we do not need the retry I can create a clean new PR

Reply:
I know :D. @bjoern-flagbit ok if I'm understanding right, do you mean current code is totally fine if you run `plugin:sync` after `plugin:refresh` from outside this command? I think the fix is nice but is it really needed? Maybe it'd be better just to remove the retry. I think it is a better option and the fix should be part of the deployment recipe where this is being used.

Reply:
Then I need to create a new PR or clean up this PR and remove plugin:refresh from here

Reply:
I'd extend as well the readme:


```
bin/console config:sync # synchronizes configuration values as defined in config/config.yaml
bin/console plugin:refresh # ensure plugins classes are loaded
bin/console plugin:sync # synchronizes plugin enable/disable status as defined in config/plugins.php 
```